Sources/StreamChat/Workers/Background/MissingEventsPublisher.swift

```diff
@@ -86,6 +103,9 @@ class MissingEventsPublisher<ExtraData: ExtraDataTypes>: EventWorker {
                     self?.eventNotificationCenter.process(payload.eventPayloads)
                 case let .failure(error):
                     log.error("Internal error: Failed to fetch and reply missing events: \(error)")
```

Comment:
This is no longer a failuer. Can you please change this from `error` to `info` and also update the message?